### DISCUSSION TITLE
Shiny and manual ShinyHost.Init

### DISCUSSION THREAD

**Discussion by sandeepc24** [shinyorg/shiny#975] [Q&A] [answered]
I am using Shiny in my F# Fabulous app and I am not able to initialize ShinyHost as I don't have access to UwpPlatform.

I am initializing using following statement
```ShinyHost.Init(new UwpPlatform(```
This works fine on Android.

I have installed `Shiny.Core` in UWP app.

I have also looked at the source code and it looks like UwpPlatform class is [there](https://github.com/shinyorg/shiny/blob/935eaecb0f4e8a6583d9e151cabeb9a05052d826/src/Shiny.Core/Platforms/Uwp/UwpPlatform.cs)

**Comment by aritchie** [comment, marked as answer]
Make sure you are targeting the version of UWP that the version of Shiny.Core is.

Please note that support for UWP is currently only offered to sponsors, but UWP support is about to be pulled from Shiny altogether in 3.0.
